packages/unmock-core/src/__tests__/runner.test.ts

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+import * as path from "path";
+import { runner, Service, UnmockPackage, gen } from "..";
+import NodeBackend from "../backend";
+const { withCodes } = gen;
+
+const servicesDirectory = path.join(__dirname, "__unmock__");
+
+describe("Node.js interceptor", () => {
+  describe("with state requests in place", () => {
+    const nodeInterceptor = new NodeBackend({ servicesDirectory });
+    const unmock = new UnmockPackage(nodeInterceptor);
+    let petstore: Service;
+
+    beforeAll(() => {
+      unmock.on();
+      petstore = unmock.services.petstore;
+    });
+    afterAll(() => unmock.off());
+
+    beforeEach(() => {
+      petstore.reset();
+    });
+
+    test("runner loop works", runner(async () => {
+      petstore.state(withCodes(200));
```

Comment:
This is not even using the runner, or?

Reply:
`test("runner loop works", runner(async () => { `